website/docs/d/entity.html.markdown

```diff
@@ -52,32 +52,97 @@ resource "newrelic_nrql_alert_condition" "foo" {
   }
 }
 
-// Filter by account ID.
-// The `accountId` tag is automatically added to all entities by the platform.
+// Ignore name case
 data "newrelic_entity" "app" {
-  name = "my-app"
+  name = "mY-aPP"
+  ignore_case = true
   domain = "APM"
   type = "APPLICATION"
-  tag {
-    key = "accountID"
-    value = "12345"
-  }
 }
+```
 
-// Ignore name case
+### Example: Filter By Account ID
+
+The default behaviour of this data source is to retrieve entities matching the specified parameters (`name`, `domain`, `type`) from NerdGraph with the credentials specified in the configuration of the provider (Account ID and API Key), filter them by the account ID specified in the configuration of the provider, and return the first match. 
+
+This would mean, if no entity with the specified search parameters is found associated with the account ID in the configuration of the provider, i.e. `NEW_RELIC_ACCOUNT_ID`, an error is thrown, stating that no matching entity has been found.
+
+```hcl
+# The entity returned by this configuration would have to 
+# belong to the account_id specified in the provider 
+# configuration, i.e. NEW_RELIC_ACCOUNT_ID.
 data "newrelic_entity" "app" {
-  name = "mY-aPP"
-  ignore_case = true
+  name   = "my-app"
+  domain = "APM"
+  type   = "APPLICATION"
+}
+```
+However, in order to cater to scenarios in which it could be necessary to retrieve an entity belonging to a subaccount using the Account ID and API Key of the parent account (for instance, when entities with identical names are present in both the parent account and subaccounts, since matching entities from subaccounts too are returned by NerdGraph), the `account_id` attribute of this data source may be availed. This ensures that the account ID in the configuration of the provider, used to filter entities returned by the API is now overridden by the `account_id` specified in the configuration; i.e., in the below example, the data source would now return an entity matching the specified `name`, belonging to the account with `account_id`.
+```hcl
+# The entity returned by this configuration, unlike in 
+# the above example, would have to belong to the account_id 
+# specified in the configuration below, i.e. 654321.
+data "newrelic_entity" "app" {
+  name       = "my-app"
+  account_id = 654321
+  domain     = "APM"
+  type       = "APPLICATION"
+}
+```
+The following example explains a use case along the lines of the aforementioned; using the `account_id` argument in the data source to allow the filtering criteria to be the `account_id` specified (of the subaccount), and not the Account ID in the provider configuration. 
```

Comment:
`account ID` You write it lowercase everywhere else

Reply:
or change it to `Account ID` in the text above. (line 66)

Reply:
done; replace `Account ID` with `account ID` everywhere.